pipelines/mining/cytomining.wdl

```diff
@@ -14,6 +14,19 @@ task profiling {
     # Input files
     String cellprofiler_analysis_directory_gsurl
     String plate_id
+    # Optional: If the CellProfiler analysis results are in an S3 bucket, this workflow can read the files directly from AWS.
```

Comment:
Does the `gsutil rsync` command on line 128 work if `cellprofiler_analysis_directory` is an S3 bucket?

Reply:
Yes, if only a `.aws/credentials` file is needed. Some scenarios also require a `.aws/config` file and this does not yet accommodate that requirement. 

@carmendv we are still testing this, which is why I have not yet sent the pull request myself. Also, the documentation needs to have a few more details added. Please feel free to try it out, but please wait to hear from me before merging this. Thank you!

Reply:
Thanks @deflaux We'll do 👍